rocky-raft/src/rocky/raft/scripts/SuperClient.java

```diff
@@ -26,22 +28,32 @@ public static void main(String[] args) throws Exception {
                 }
                 break;
             case "p":
-                superClient.doPost(config, args[1]);
+                try {
+                    superClient.doPost(config, args[1], Long.parseLong(args[2]));
```

Comment:
We shouldn't require user to supply id. SuperClient should generate UUID and retry internally.